sourcetools/j9constantpool/CMakeLists.txt

```diff
@@ -20,6 +20,20 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ################################################################################
 
+# CMake doesn't properly handle directory separators when building in cygwin
+# but using a windows java binary. IE it uses ':' as a directory separator,
+# rather than required ';'
+
+# WIN32 will be true when the build is targeting windows, but CMAKE_HOST_WIN32 is false on cygwin
+if(WIN32 AND NOT CMAKE_HOST_WIN32)
+	# put  "-cp <Path_to_objectmodel_jar>" into a text file we will pass in with '@file'
+	file(GENERATE OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/cygwin_classpath" CONTENT "-cp\n$<TARGET_PROPERTY:objectmodel,JAR_FILE>\n")
```

Comment:
Is there supposed to be `\n` after `-cp`?

Reply:
yes. The syntax for `@` files is to have arguments separated by newlines, rather than spaces